### ISSUE TITLE
TypeError: 'NoneType' object is not callable

### ISSUE THREAD

**Issue by leluchbritansky** [RicterZ/nhentai#133]
[11:31:34] [INFO] Fetching doujinshi information of id 222111
Traceback (most recent call last):
  File "/usr/local/bin/nhentai", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/nhentai/command.py", line 83, in main
    doujinshi_info = doujinshi_parser(id_)
  File "/usr/local/lib/python2.7/dist-packages/nhentai/parser.py", line 196, in doujinshi_parser
    field.find_all('a', attrs={'class': 'tag'})]
TypeError: 'NoneType' object is not callable

**OR**

[11:50:06] [INFO] Fetching doujinshi information of id 222111
Traceback (most recent call last):
  File "/usr/local/bin/nhentai", line 11, in <module>
    load_entry_point('nhentai==0.3.9', 'console_scripts', 'nhentai')()
  File "/usr/local/lib/python3.8/dist-packages/nhentai-0.3.9-py3.8.egg/nhentai/command.py", line 83, in main
    doujinshi_info = doujinshi_parser(id_)
  File "/usr/local/lib/python3.8/dist-packages/nhentai-0.3.9-py3.8.egg/nhentai/parser.py", line 195, in doujinshi_parser
    data = [sub_field.contents[0].strip() for sub_field in
  File "/usr/local/lib/python3.8/dist-packages/nhentai-0.3.9-py3.8.egg/nhentai/parser.py", line 195, in <listcomp>
    data = [sub_field.contents[0].strip() for sub_field in
TypeError: 'NoneType' object is not callable

**What I'm doing wrong?**

**Comment by ODtian** [comment]
Same issue.
```
Traceback (most recent call last):
  File "/usr/local/bin/nhentai", line 11, in <module>
    load_entry_point('nhentai==0.3.9', 'console_scripts', 'nhentai')()
  File "/usr/local/lib/python3.8/site-packages/nhentai/command.py", line 83, in main
    doujinshi_info = doujinshi_parser(id_)
  File "/usr/local/lib/python3.8/site-packages/nhentai/parser.py", line 195, in doujinshi_parser
    data = [sub_field.contents[0].strip() for sub_field in
  File "/usr/local/lib/python3.8/site-packages/nhentai/parser.py", line 195, in <listcomp>
    data = [sub_field.contents[0].strip() for sub_field in
TypeError: 'NoneType' object is not callable
```

**Comment by git-horst** [comment]
Same issue. Hope it be fixed :)

**Comment by symant233** [comment]
Thanks, #134 helps!

**Comment by RicterZ** [comment]
fixed on 0.4.0
